base_layer/wallet_ffi/src/lib.rs

```diff
@@ -3952,6 +3952,48 @@ pub unsafe extern "C" fn wallet_sync_with_base_node(wallet: *mut TariWallet, err
     }
 }
 
+/// This function will tell the wallet to do a coin split.
+///
+/// ## Arguments
+/// `wallet` - The TariWallet pointer
+/// `amount` - The amount to split
+/// `count` - The number of times to split the amount
+/// `fee` - The transaction fee
+/// `lock_height` - The number of bocks to lock the transaction for
+/// `error_out` - Pointer to an int which will be modified to an error code should one occur, may not be null. Functions
+/// as an out parameter.
+///
+/// ## Returns
+/// `()` - Does not return a value, equivalent to void in C.
+///
+/// # Safety
+/// None
+#[no_mangle]
+pub unsafe extern "C" fn wallet_coin_spit(
```

Comment:
```suggestion
pub unsafe extern "C" fn wallet_coin_split(
```

Reply:
Thanks for picking up this spelling error.